lib/catalog/martyrology.ts

```diff
@@ -3321,7 +3323,10 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       titles: [Title.Religious],
     },
     passion_of_saint_john_the_baptist: {
+      // Note: unlike the celebration of the Nativity of Saint John the Baptist, here his title of Martyr is indeed indicated.
```

Comment:
Just a nitpick.

/cc @TobiTenno, maybe this could be written more naturally / better. Should we use `specified` instead of `indicated`?


```suggestion
      // Note: Unlike the celebration of the Nativity of Saint John the Baptist, here the title of a martyr must be indicated.
```

Reply:
In this context, I believe "indicated" could mostly mean "highly recommended," whereas "specified" has more of a connotation of specs. So indeed, "specified" would be more appropriate here. But yes, this is really nitpicking!

If you see other things like this, feel free to open PRs and suggest improvements (teamwork!). On my side, unless I'm writing something completely wrong or writing incomprehensible things (English is not my native language), I prefer to focus on functional improvements to Romcal. Otherwise, I won't have time to do more on this project. 😄

Reply:
I am nitpicking about the semantics! :wink: Sorry! :sweat_smile:

---

From my POV, the celebration definitions in calendars (which have the `LiturgicalDayInput` type) are composed of the celebration ID (a property of the `Inputs`) and the props of `LiturgicalDayInput` are the _specs_ (as you sort of called it) or _metadata_ (as I call it). Thus, from my POV, they are specified, or defined for a particular celebration inscribed in a calendar. :wink:

> If you see other things like this, feel free to open PRs and suggest improvements (teamwork!).

Yeah, I will, however, unfortunately, I’ve been really busy lately, unfortunately. I wish I could find more time for romcal.

> On my side, unless I'm writing something completely wrong or writing incomprehensible things (English is not my native language), I prefer to focus on functional improvements to Romcal. Otherwise, I won't have time to do more on this project. 😇

I get it, Etienne. Thank you very much for you do for romcal! :pray:

Neither am I a native English speaker, however, I tried to do my best (as you do). :wink:

I agree the functional improvements are more important that this, that is why I label the less imporant (PR) comments as nitpicks: they are not necessary to be implemented/fixed. They could be completely ignored, however, I still want to raise them. Sorry about the noise. :wink:

Reply:
i generally agree w/ @emagnier on this one. titles are personal, not ephemerally disassociated from merit given a saint, so the possessive "his title of Martyr" is the most natural in English grammar.

Reply:
I think the best resolution for the future on these is to include metadata/specifications/whatever you wanna call them as an object field, like "sources" so that we can reference the sources programmatically and enforce them using linting or typing

Reply:
Including a way to indicate rules/sources is an important topic that we need to explore 💯.  And having linting or typing would be very cool!

We also need to keep some flexibility for special cases like this one. In this case, the main issue is the lack of an explicit mention in the Roman Missal. Especially considering that the Missal's rules specify that only martyrs are commemorated in red, along with a few other clearly mentioned celebrations, but without mentioning the Passion of Saint John the Baptist. The title "Martyr" is not explicitly given for this commemoration (at least in Latin and English, but in French, that's the case), and we don't want to add it. However, we know that in practice, this commemoration is indeed celebrated in red.

I didn't think there'd be so much talk when I opened this PR 😄. But we're getting into some interesting topics 👍